src/latexify/codegen/function_codegen.py

```diff
@@ -355,6 +355,30 @@ def visit_Call(self, node: ast.Call) -> str:
             (r"\mathrm{" + func_str + r"}\left(", r"\right)"),
         )
 
+        if func_str in ("ndarray", "array"):
+            arg = node.args[0]
+            if not isinstance(arg, ast.List) or not arg.elts:
+                return None
+
+            row0 = arg.elts[0]
+
+            if not isinstance(row0, ast.List):
+                return self._generate_ndarray([self.visit(x) for x in arg.elts])
+
+            if not row0.elts:
+                return None
+
+            nCols = len(row0.elts)
```

Comment:
nCols is not allowed under the naming convention.
https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations
```suggestion
            ncols = len(row0.elts)
```